src/components/emote-picker-on-input/EmotePickerOnInput.jsx

```diff
@@ -1,15 +1,13 @@
 import React, { useState } from 'react';
-import Popup from 'reactjs-popup';
-import 'reactjs-popup/dist/index.css';
 
 import EmotePicker from '../emote-picker/EmotePicker';
```

Comment:
The dependency `reactjs-popup` can be removed right?

Reply:
The removal will happen in the next PR where I update all the dependencies. 🥸